docs/admin/notifications.rst

```diff
@@ -61,6 +61,51 @@ please refer to the `Open Zaak authorization documentation`_.
 For more information on how to set up subscriptions in Open Notificaties, please refer
 to the `Open Notificaties subscription documentation`_.
 
+Automatic retry for notifications
+=================================
+
+By default, sending notifications has automatic retry behaviour, i.e. if the notification
+publishing task has failed, it will automatically be scheduled/tried again until the maximum
```

Comment:
```suggestion
publishing task has failed, it will automatically be rescheduled/tried again until the maximum
```
Scheduled and tried again do not sound interchangeable to me. Either rescheduled or _and_ instead of / sounds more correct.